build.gradle

```diff
@@ -80,7 +81,7 @@ githubRelease {
    prerelease = false
    overwrite = false
    generateReleaseNotes = false
-   body = layout.projectDirectory.file('release/changes.md').asFile.text.trim()
+   body = releaseNotes().get()
```

Comment:
`githubRelease.body` is actually a provider so you don't need to call `.get()` here.

Reply:
Fixed, thanks.